integration/dbt/scripts/dbt-ol

```diff
@@ -152,13 +152,19 @@ def main():
 
     pre_run_time = time.time()
     # Execute dbt in external process
-    process = subprocess.Popen(["dbt"] + sys.argv[1:], stdout=sys.stdout, stderr=sys.stderr)
-    return_code = process.wait()
+
+    force_send_events = "send-events" in sys.argv[1:]
```

Comment:
I would force `send-events` to be first argument in this case? It doesn't have to be the only one, e.g. it still may be needed to pass `--target` as argument to know which profile should be used.